backend/monitoring/dashboard.py

```
@@ -1611,25 +1629,29 @@ async def get_live_audio_video_sessions():
+                    "session_id": str(session['session_id']),
                     "user_id": session['user_id'],
                     "service_type": session['service_type'],
                     "status": session['status'],
                     "start_time": session['start_time'].isoformat() if session['start_time'] else None,
                     "end_time": session['end_time'].isoformat() if session['end_time'] else None,
-                    "duration": session['duration'],
-                    "cost": float(session['cost']) if session['cost'] else 0.0,
-                    "agora_channel": session['agora_channel']
+                    "duration": session['duration_minutes'],
+                    "cost": float(session['credits_used'] * 0.1) if session['credits_used'] else 0.0,
+                    "agora_channel": session.get('agora_channel') or self._extract_agora_channel_from_session_data(session.get('session_data'))
```

Ruff (0.12.2)

Undefined name `self`

(F821)